core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/base/parameter/stack/AwsStackV4Parameters.java

```diff
@@ -6,20 +6,40 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.sequenceiq.cloudbreak.cloud.PlatformParametersConsts;
 import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
 
 import io.swagger.annotations.ApiModel;
 import io.swagger.annotations.ApiModelProperty;
 
+import java.util.Map;
+
 @ApiModel
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonInclude(Include.NON_NULL)
 public class AwsStackV4Parameters extends StackV4ParameterBase {
+    @ApiModelProperty
+    private boolean createCloudwatch;
```

Comment:
You have added a new model to `StackV4Request`. This modifies the final API of the datalake service. You should modify the tests to include this.